packages/CodingStandard/src/Fixer/Property/ArrayPropertyDefaultValueFixer.php

```diff
@@ -0,0 +1,77 @@
+<?php declare(strict_types=1);
+
+namespace Symplify\CodingStandard\Fixer\Property;
+
+use Nette\Utils\Strings;
+use PhpCsFixer\AbstractFixer;
+use PhpCsFixer\DocBlock\DocBlock;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+use PhpCsFixer\FixerDefinition\FixerDefinitionInterface;
+use PhpCsFixer\Tokenizer\Token;
+use PhpCsFixer\Tokenizer\Tokens;
+use SplFileInfo;
+
+final class ArrayPropertyDefaultValueFixer extends AbstractFixer
```

Comment:
I would recommend implementing `FixerInterface` and not extend the internal abstract. The interface is not that complex and I'm happy to help out out it if needed. 
Also; we keep a BC promise on the interface, which we do not for internal classes. So its the safer bet.